utils/util.js

```diff
@@ -0,0 +1,7 @@
+const useHandleScroll = (elemRef, setterFunc) => {
```

Comment:
The name of the hook is ambiguous and also not placed correctly. We are keeping hooks at lib/hooks/*

Reply:
It has been taken care.